src/parallax/fourth/sections/one.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./one.module.scss";
+import btnStyles from "../button.module.scss";
+
+const Section1 = () => {
+  return (
+    <section className={styles.one}>
+      <div className={styles.left}>
+        <h1>
+          Air <br />
+          balloon <br />
+          flights
+        </h1>
+        <p>
+          Some text saying something about how awesome air balloon flights are
+          and that you should try it sometime, or maybe something else idk
```

Comment:
it's awesome indeed, i've done it once at 4am to see the sunrise at 5am and it was magical